docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  web-cors-proxy:
+    container_name: cors-anywhere
+    image: cors-anywhere:latest
+    command: 'sh -c "PORT=4800 yarn start"'
+    build:
+      context: .
+      dockerfile: Dockerfile
+    stdin_open: true
+    tty: true
+    ports:
+      - "4800:4800"
```

Comment:
This port will need to be mapped to the one on the mimic server.  Currently the copilot is running there.  Here's the deets:
<img width="1189" alt="image" src="https://github.com/user-attachments/assets/58c91d53-aa49-41b9-8539-2a409448c404">

From the copilot docker-compose.yml:
```yaml
    ports:
      - "8080:7059"
```
So I think this needs to be  "8080:4800" in this case.